lib/gui/modules/analytics.js

```diff
@@ -142,13 +142,15 @@ analytics.service('AnalyticsService', function($log, $window, $mixpanel, Setting
    */
   this.logEvent = (message, data) => {
     const flatStartCaseData = utils.makeFlatStartCaseObject(data);
+    const anonymizedFlatStartCaseData = utils.hideObjectAbsolutePaths(flatStartCaseData);
```

Comment:
I wonder if `anonymizedFlatStartCaseData` could be given a nicer name? It's a bit of a mouthful! ;-)